ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -30,6 +28,18 @@ tag_specification:
 test_binary_build_commands: |
   make build-openshift-ci-test-bin
 tests:
+- as: lint
+  commands: export GOLANGCI_LINT_CACHE=/tmp/.cache make lint
+  container:
+    from: test-bin
+- as: format
```

Comment:
ended up removing the `format` check. If people complain about not knowing why `format-check` fails....then we can add it later.